experiments/run_finetuning.py

```diff
@@ -62,6 +62,21 @@ def get_train_dataloader(self) -> DataLoader:
 
     def get_eval_dataloader(self, _) -> DataLoader:
         return self.test_loader
+    
+    def compute_loss(self, model, inputs, return_outputs=False):
+        labels = inputs.pop('labels')
+        attention_mask = inputs["attention_mask"]
+        outputs = model(**inputs)
+        labels = labels[..., 1:].contiguous()
```

Comment:
Nit: we should double check whether `contiguous` is required here, as we don't use it in our ppl implementation.